.github/workflows/sync-branch.yml

```diff
@@ -1,16 +1,44 @@
+# This workflow is used to create PRs to merge changes from the main branch to the next branch. It's mostly a front-end
+# to `flub merge branches`, which is where the source code can be found. This also means you can replicate this workflow
+# manually by running `flub merge branches` locally and providing a PAT.
+#
+# When run, the workflow executes `flub merge branches`, which first checks if a main/next merge PR is already open. If
+# it is, the workflow exits with no error.
+#
+# If there is no PR, then the tool will open one by following this logic
+#
+# - Find the commits that need to be merged into next.
+# - Take a batch of commits and iterate over them.
+# - Test each commit for mergeability into next.
+# - If all commits merge cleanly, open a PR at the last commit in the batch.
+# - If a commit cannot be merged cleanly, open a PR at the _previous_ commit.
+# - Once that PR is merged, then the next time the workflow runs, it will detect the conflicting commit and open a PR
+#   with that commit only.
+# 
+# Each time the workflow runs, the tool will determine the commits to merge based entirely on the git repo state.
+
 name: Sync branches
 
 on:
   schedule:
-    - cron: "0 */2 * * 1-5" # Cron timezone is in UTC
+    # At 0 minutes past the hour, every 2 hours, Monday through Friday. Cron timezone is in UTC.
+    - cron: "0 */2 * * 1-5"
```

Comment:
```suggestion
    # At 0 minutes past the hour, every 1 hour, Monday through Friday. Cron timezone is in UTC.
    - cron: "0 */1 * * 1-5"
```

Reply:
I think hourly is better than every two.

Also, maybe we could trigger on commits to next... then every time a PR is merged into next the workflow will run immediately.

Reply:
Will leave these for separate changes